app/src/main/java/org/mozilla/fenix/home/HomeMenuBuilder.kt

```diff
@@ -114,6 +115,18 @@ class HomeMenuBuilder(
                     }
                 )
             }
+            HomeMenu.Item.ManageAccountAndDevices -> {
+                homeActivity.openToBrowserAndLoad(
+                    searchTermOrURL =
+                    if (Config.channel.isMozillaOnline && context.settings().allowDomesticChinaFxaServer) {
```

Comment:
If we use the `Config.channel.isMozillaOnline` check when we build the menu maybe here is not needed again.